doc/netplan-get.md

```diff
@@ -1,39 +1,39 @@
 ---
-title: netplan-get
+title: NETPLAN-GET
 section: 8
 author:
 - Lukas Märdian (lukas.maerdian@canonical.com)
 ...
 
 ## NAME
 
-netplan-get - read merged netplan YAML configuration
+`netplan-get` - read merged Netplan YAML configuration
 
 ## SYNOPSIS
 
-  **netplan** [--debug] **get** -h | --help
+  **`netplan`** \[*--debug*\] **get** **-h**|**--help**
 
-  **netplan** [--debug] **get** [--root-dir=ROOT_DIR] [key]
+  **`netplan`** \[*--debug*\] **get** \[*--root-dir=ROOT_DIR*\] \[*key*\]
 
 ## DESCRIPTION
 
-**netplan get [key]** reads all YAML files from ``/{etc,lib,run}/netplan/*.yaml`` and returns a merged view of the current configuration
+**`netplan get \[*key*\]`** reads all YAML files from `/{etc,lib,run}/netplan/*.yaml` and returns a merged view of the current configuration.
```

Comment:
The back slashes and asterisks here are being included in the text as they are inside ` `` `

Reply:
Thanks. Oversight. Fixed.